ee/src/main/java/org/jboss/as/ee/subsystem/DefaultBindingsAdd.java

```diff
@@ -36,6 +36,8 @@
  */
 public class DefaultBindingsAdd extends AbstractBoottimeAddStepHandler {
 
+    public static final int STRUCTURE_EE_DEFAULT_BINDINGS_CONFIG = 0x1B01;
```

Comment:
Instead of a temporary change that needs cleaning up we should just aim to get the WildFly Core change through and use it after the next tag.